_data/shoebkhan30/shoebkhan30.md

```diff
@@ -0,0 +1,7 @@
+---
+name: SHOEB KHAN 
+institution: SHRI VAISHNAV VIDHYAPEETH VISHWAVIDHYALA 🚩 
+profile_pic: me.jpg 
+quote: The only time to buy these is on a day with no “y” in it.
```

Comment:
Remove the double-quotes as they are non-ascii characters.

Reply:
Done. Removed them.